Makefile

```
@@ -950,26 +947,51 @@ _builtin_docs_validate:
 
 _builtin_release_status: _builtin_require_environment
 	@$(UV) lock --project "$(PROJECT_ROOT)" --check
 	@git -C "$(PROJECT_ROOT)" diff --quiet
 	@git -C "$(PROJECT_ROOT)" diff --cached --quiet
 
+# Release orchestration belongs to the verb that already owns releases. It
+# previously lived in the workspace-root custom.mk as _custom_release_rel,
+# putting the release pipeline outside the Make monopoly and making Release CI
+# depend on a hand-written surface. Knobs come from the caller environment so
+# no new selector variables enter the public surface; PUSH=1 opts into pushing.
+_builtin_release_rel: _builtin_require_environment
```

checkmake (0.3.2)

[warning] Target body for "_builtin_release_rel" exceeds allowed length of 5 lines (17).

(maxbodylength)